core/server/common/src/main/java/alluxio/master/journal/raft/RaftPrimarySelector.java

```diff
@@ -29,33 +23,13 @@
  */
 @ThreadSafe
 public class RaftPrimarySelector extends AbstractPrimarySelector {
```

Comment:
Can you update some of the comments in this file that still say copycat?

Reply:
Updated.